docs/book/src/reference/versions.md

```diff
@@ -144,6 +144,9 @@ The Kubeadm Control Plane requires the Kubeadm Bootstrap Provider.
 | v1.3 (v1beta1)       | v1.10.0                         |
 | >= v1.3.4 (v1beta1)  | v1.10.1                         |
 | v1.4 (v1beta1)       | v1.10.1                         |
+| >= v1.4.6 (v1beta1)  | v1.11.1                         |
+| v1.5 (v1beta1)       | v1.10.1                         |
```

Comment:
I overlooked it in other PR, is this correct?

Reply:
Yes it is correct

Reply:
Oure CoreDNS support in this table depends on the migration library we use

Reply:
You're welcome. No problem :) Thx for asking

Reply:
Just assuming PR authors know what they're doing doesn't help much on PRs :joy: